pyproject.toml

```diff
@@ -89,11 +89,6 @@ src="gooddata-*-client/gooddata_*_client/configuration.py"
 # clients __init__.py
 src="gooddata-*-client/gooddata_*_client/__init__.py"
 
-[[tool.tbump.file]]
```

Comment:
Pleas bump version in:
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-afm-client.yaml#L9
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-metadata-client.yaml#L9
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-scan-client.yaml#L9

I forgot to update bump configuration when I did the change of `generate_client.sh` script.